webdash/task.html

```diff
@@ -12,138 +12,149 @@ <h5 style="text-align: center">Not Found</h5>
   </div>
 
   <div ng-show="loaded">
-  <table class="task-desc-table">
```

Comment:
What happened here? I thought you were just adding a section?

Reply:
The indentation was off in the file which is why the change looks so large.